ws-nextjs-app/pages/[service]/send/[id]/FormField/File/styles.ts

```diff
@@ -0,0 +1,110 @@
+import pixelsToRem from '#app/utilities/pixelsToRem';
+import { Theme, css } from '@emotion/react';
+
+export default {
+  fileInput: () =>
+    css({
+      display: 'none',
+    }),
+  fileUploadButton: ({ spacings, palette, fontVariants, fontSizes }: Theme) => {
+    const topBottomPadding = `${spacings.HALF + spacings.FULL}rem`;
+    const leftRightPadding = `${spacings.DOUBLE}rem`;
+
+    return css({
+      padding: `${topBottomPadding} ${leftRightPadding} ${topBottomPadding} ${leftRightPadding}`,
+      border: 'none',
+      backgroundColor: '#0071F1',
```

Comment:
Think we can use the new Theme colours for this and the hover/focus state:
```
export const SERVICE_NEUTRAL_CORE = '#0071F1';
export const SERVICE_NEUTRAL_DARK = '#0051AD';
```

Reply:
Good spot, done.